crates/stdsimd-test/src/lib.rs

```diff
@@ -46,38 +45,56 @@ lazy_static! {
     static ref DISASSEMBLY: HashMap<String, Vec<Function>> = disassemble_myself();
 }
 
+#[derive(Debug)]
 struct Function {
     addr: Option<usize>,
     instrs: Vec<Instruction>,
 }
 
+#[derive(Debug)]
 struct Instruction {
     parts: Vec<String>,
 }
 
-fn normalize(symbol: &str) -> String {
+fn normalize(mut symbol: &str) -> String {
```

Comment:
TODO: document what this does - in a nutshell, it brings the shim function names from potentially mangled names with paths, etc. down to the `stdsimd_test_shim_...` form, so that we can query these easily by doing HashSet lookup.